.travis.yml

```diff
@@ -59,6 +62,31 @@ jobs:
       script:
         - bash .travis.sh script
         - bash .travis.sh test
+    - name: azure
+      env:
+        - K8_PROVIDER=azure
+        - CCO_INTERACTIVE_CI=interactive.yaml
+        - HELM_VERSION=v2.16.1
+        - KUBECONFIG=kubeconfig_terraform-cco
+      before_install: pip install -e .
+      install: bash .travis.sh install-tools
+      script:
+        - az login --service-principal -u $ARM_CLIENT_ID -p $ARM_CLIENT_SECRET --tenant $ARM_TENANT_ID
+        - cd terraform/azure &&
+          terraform init -input=false &&
+          terraform validate &&
+          terraform apply -input=false -auto-approve &&
+          terraform output cco-interactive-yaml > interactive.yaml &&
+          terraform output kube_config > kubeconfig_terraform-cco
+        - kubectl get pods --all-namespaces
+        - echo " " | ckan-cloud-operator cluster initialize --interactive --cluster-provider=azure
+        - kubectl get ns && kubectl get pods -n ckan-cloud
+        - ckan-cloud-operator ckan instance create helm --instance-id a-ckan-instance --instance-name a-ckan-instance --update ../../sample-values/values.azure.yaml
+        - kubectl -n a-ckan-instance describe pods
+      after_success:
+        - terraform destroy -auto-approve
```

Comment:
check out the destroy_aws.sh script - it does a bit more than terraform destroy (e.g. free up volumes, load balancers, buckets) so that we're not left with a lot of old resources in our account.